pkg/oc/admin/policy/modify_roles.go

```diff
@@ -320,6 +321,13 @@ func (o *RoleModificationOptions) CompleteUserWithSA(f *clientcmd.Factory, cmd *
 		return errors.New("you must specify at least one user or service account")
 	}
 
+	// return an error if a fully-qualified service-account name is used
```

Comment:
You should instead check that all names are valid SA names.

Reply:
Thanks, kept this check and added an additional check for the overall validity of each given SA name